arroyo-connectors/src/lib.rs

```diff
@@ -178,7 +178,8 @@ impl<C: Connector> ErasedConnector for C {
         Ok(self.config_description(self.parse_config(s)?))
     }
 
-    fn validate_config(&self, _: &serde_json::Value) -> Result<(), serde_json::Error> {
+    fn validate_config(&self, config: &serde_json::Value) -> Result<(), serde_json::Error> {
+        self.parse_config(config)?;
```

Comment:
This change needs to be made for validate_table as well — looks like we lost the validation logic in the refactor

Reply:
Right, done.